mobify/progressive-web-scaffold
# Filter list of available regions in shipping page

Also display an input box if no regions are available to display.

## Changes
- Adds a set of new selectors to retrieve both the available regions, as well as the currently selected Country (id)
- Using those selectors, add a new selector that provides a filtered list of regions (based on the selected country id)
- Conditionally display input box with `region` name if no regions are available to show (also update submission action to include this form value)

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add item to cart and visit `/checkout/cart/`
- Check out to proceed to shipping page
- Note that on initial load, only US states are shown
- Selecting a country like `Canada` will update list with provinces, while selecting something like `Cameroon` will display an empty input field
